### ISSUE TITLE
Cast node TypeError

### ISSUE THREAD

**Issue by liamhogan** [Hypnos3/node-red-contrib-cast#37]
I'm using node-red and am unable to cast anything because the cast node fails to deploy. When I install the package, I'm getting an error when adding the node: 
> node-red-contrib-cast:cast-to-client : TypeError: ProtoBuf.loadProtoFile is not a function

When I try to deploy a flow which contains the cast node, I get the error:
> Waiting for missing types to be registered: cast-to-client 

Any ideas?

**Comment by liamhogan** [comment]
Edit: I've discovered that this issue is related to the installed version of protobufjs. 
I had the latest version (6.8.8) which seemed to be returning the error in my previous post, but when I downgraded to 5.0.3, cast works as intended without error. 

I have seen a couple of issues reported for other packages where v6+ of protobufjs has been the cause of trouble which is why I tried the downgrade. I don't have enough in-depth knowledge of the code to know why that is an issue here but seems worth investigating.
